### DISCUSSION TITLE
How can I add svg animated html file to my logo ?

### DISCUSSION THREAD

**Discussion by akhilsahuji** [jpanther/congo#237] [Q&A]
I am trying to change the logo to an animated SVG. I am confused about how I can add a logo.html in this theme.
I want to add the below SVG to the logo  but with animation 

![Group 1 (3)](https://user-images.githubusercontent.com/46328560/176824540-bfbd14d9-7273-4c3d-ab00-8ca6fe4a10fc.svg)

I tried to add logo.html to my assets/img and tried to call `logo = "img/logo.jpg"` 
but it breaks the code.
Any suggestion would be appreciated.

**Comment by jpanther** [comment]
The logo currently only supports images that can be displayed in an HTML `img` tag. Although SVG assets are not currently supported, you can provide a custom header partial and replace the normal logo handling logic with your own SVG markup.

**Reply by akhilsahuji** [reply]
So, do I have to make a separate header file or do I have to edit the hamburger.html?

**Reply by jpanther** [reply]
Quickest way would be to copy the hamburger.html file into your project as `/partials/header/custom.html` and then in your config you can use `header.layout = "custom"` and it will reference your local copy of the file. This way when you upgrade the theme version in future it wont overwrite your customisations.

**Reply by akhilsahuji** [reply]
ok, will try and give an update.
thanks for the suggestion 😄 .
